dev/make-distribution.sh

```diff
@@ -108,7 +108,7 @@ fi
 
 if [ -z "$JAVA_HOME" ]; then
   echo "Error: JAVA_HOME is not set, cannot proceed."
-  exit -1
+  exit 255
```

Comment:
SC2242 (error): Can only exit with status 0-255. Other data should be written to stdout/stderr.